app/db/models/lawpolicy/metadata.py

```diff
@@ -0,0 +1,37 @@
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from app.db.models.app.users import Organisation
+from .family import Family
+
+from app.db.session import Base
+
+
+class MetadataTaxonomy(Base):
+
+    __tablename__ = "metadata_taxonomy"
+
+    id = sa.Column(sa.Integer, primary_key=True)
+    name = sa.Column(sa.Text, nullable=False)
```

Comment:
Similar comment here re. id tbh, it feels like a taxonomy name should be a unique identifier

Reply:
done